pkg/cephmgr/rgw/leader.go

```diff
@@ -104,6 +105,25 @@ func RemoveObjectStore(context *clusterd.Context) error {
 	return nil
 }
 
+func GetRGWEndpoints(etcdClient etcd.KeysAPI, clusterInventory *inventory.Config) (host, ipEndpoint string, found bool, err error) {
+	appliedNodes, err := util.GetDirChildKeys(etcdClient, getRGWNodesKey(true))
+	if err != nil {
+		return "", "", false, err
+	}
+
+	for nodeID := range appliedNodes.Iter() {
+		// just return the details of the first RGW node we can find
+		nodeDetails, ok := clusterInventory.Nodes[nodeID]
+		if ok {
+			host = getRGWEndpoint(DNSName)
+			ipEndpoint = getRGWEndpoint(nodeDetails.PublicIP)
+			return host, ipEndpoint, true, nil
```

Comment:
rgw will be started on all the nodes currently in the cluster. If we returned all of them, they would have the option of putting them behind a load balancer, or they can just pick the first one.